src/containers/paint-editor.jsx

```diff
@@ -425,8 +471,9 @@ PaintEditor.propTypes = {
     isEyeDropping: PropTypes.bool,
     mode: PropTypes.oneOf(Object.keys(Modes)).isRequired,
     name: PropTypes.string,
+    onCopyToClipboard: PropTypes.func.isRequired,
     onDeactivateEyeDropper: PropTypes.func.isRequired,
-    onKeyPress: PropTypes.func.isRequired,
+    onPasteFromClipboard: PropTypes.func.isRequired,
```

Comment:
Same comment as before, the `PaintEditor` should be wrapped in `CopyPasteHOC` before exporting, since it won't be completely functional without it.

Reply:
👍